TOOLS/lua/autocrop.lua

```diff
@@ -107,8 +128,21 @@ function do_crop()
         )
     end
     -- remove the cropdetect filter
-    del_filter_if_present(cropdetect_label)
+    remove_crop(cropdetect_label)
     timer=nil
 end
 
-mp.add_key_binding("C", "auto_crop", autocrop_start)
+function auto_corp()
+    local duration = mp.get_property_native("duration")
+    local percent = mp.get_property_native("percent-pos")
+    if options.auto == "yes"
+        and not vf_exist(crop_label)
+        and percent < options.auto_percent
+        and duration > options.auto_duration
```

Comment:
In most cases, the movie will have a stable black border, and the general video only has a black border at the beginning.  Such a long `duration` is to avoid misjudgment in that.

I set `save-position-on-quit` in mpv.conf.
If I manually disable the "crop" filter, the playback progress reaches a large black border screen, and then I reopen the video, "autocrop" will cause the screen to be cropped incorrectly